ci-operator/step-registry/cert-manager/custom-apiserver-cert/cert-manager-custom-apiserver-cert-commands.sh

```diff
@@ -189,26 +182,44 @@ INTERVAL=30
 COUNTER=0
 while :;
 do
-    echo "Checking clusteroperator kube-apiserver status for the #${COUNTER}-th time ..."
+    echo "Checking if clusteroperator kube-apiserver rollout finished for the #${COUNTER}-th time ..."
     if [ "$(oc get --no-headers clusteroperator kube-apiserver | awk '{print $3 $4 $5}')" == TrueFalseFalse ]; then
-        echo 'The clusteroperator kube-apiserver status becomes "True False False".' && break
+        echo 'The clusteroperator kube-apiserver status becomes "True False False", indicates rollout finished.' && break
     fi
     ((++COUNTER))
     if [[ $COUNTER -eq $MAX_RETRY ]]; then
         echo "The clusteroperator kube-apiserver status is not ready after $((MAX_RETRY * INTERVAL)) seconds. Dumping status:"
-        oc get clusteroperator kube-apiserver
+        oc get clusteroperator kube-apiserver -o=jsonpath='{.status}'
         exit 1
     fi
     sleep $INTERVAL
 done
 
 echo "Validating the cert-manager customized Apiserver serving certificate."
-CURL_OUTPUT=$(curl -IsS --cacert $CA_FILE --connect-timeout 30 "https://$NEW_API_FQDN:6443" 2>&1 || true)
-if [[ ! "$CURL_OUTPUT" =~ "HTTP/2 403" ]]; then
-    echo -e "Fails to validate the cert-manager customized Apiserver serving certificate. Dumping the curl output:\n${CURL_OUTPUT}"
-    exit 1
+# The CA_FILE will also be used later to update KUBECONFIG
+oc extract secret/cert-manager-managed-alt-api-tls -n openshift-config
```

Comment:
Just note:
Line 168 (of original file) shows: `Note, if $NEW_API_FQDN is $ORIGINAL_API_FQDN other than an alternative FQDN, all oc commands afterwards need to add the --insecure-skip-tls-verify flag before the KUBECONFIG is updated later`.

This means if we in future covers the situation of "$NEW_API_FQDN is $ORIGINAL_API_FQDN", in that situation, `oc extract ...` here will fail unless adding `--insecure-skip-tls-verify`. This is why the `oc extract ...` was put before `oc patch apiserver cluster ...`.
Feel free to keep original `oc extract ...` without moving it, but you can add a comment:
```
# Note, if $NEW_API_FQDN is $ORIGINAL_API_FQDN other than an alternative FQDN, all oc commands afterwards need to add the --insecure-skip-tls-verify flag before the KUBECONFIG is updated later. So `oc extract ...` is done in advance.
```
Or go ahead with your change, but you can add a comment as reminder:
```
# Note, if $NEW_API_FQDN is $ORIGINAL_API_FQDN other than an alternative FQDN, oc commands like `oc extract ...` need to add the --insecure-skip-tls-verify flag before the KUBECONFIG is updated later
```